BossMod/AI/AIBehaviour.cs

```diff
@@ -36,7 +36,8 @@ public void Execute(Actor player, Actor master)
                 return;
 
             // keep master in focus
-            FocusMaster(master);
+            if (Plugin._ai._config.FocusTargetLeader)
```

Comment:
Config is accessible directly via Service.Config.Get if needed, was this the only reason for making the _ai field static?